common/changes/@typespec/json-schema/json-schema_2023-04-26-21-59.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@typespec/json-schema",
+      "comment": "Add @typespec/json-schema for defining and emitting TypeSpec to standard JSON Schema",
+      "type": "minor"
```

Comment:
Should this package also be pinned to the rest of the packages versions?

Reply:
Yup!